tests/bson/bug0974-001.phpt

```diff
@@ -44,7 +44,7 @@ object(stdClass)#%d (%d) {
   }
 }
 object(stdClass)#3 (2) {
```

Comment:
While you're in this file, `#3 (2)` above should probably be `#%d (%d)`. The number in parentheses is not so important and we can assert the number of fields, but we definitely don't want to expect object sequences.

If you want to take a pass through all the tests looking for `)#\d` that might turn up more things we missed.

Reply:
Removed a number of occurrences across tests.